tools/generate_rust_api.py

```
@@ -210,28 +227,79 @@ def resolve_multiparm(name: str) -> MultiparmInfo:
+        or not p_data.get("menu_labels")
+    ):
+        return None
+
+    base_enum_name = pascal_case(p_name)
+    unique_enum_name = enum_resolver.resolve(base_enum_name)
+    enum_name = f"{struct_name}{unique_enum_name}"
+
+    variants = []
+    variant_resolver = SuffixResolver(separator="")
+
+    for i, (tok, lab) in enumerate(zip(p_data["menu_items"], p_data["menu_labels"])):
```

Ruff (0.15.7)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)